libspawner/inc/win32/runner.h

```diff
@@ -15,9 +15,6 @@
 class runner : public base_runner {
     mutex_c suspend_mutex_;
 
-    void copy_environment(TCHAR* dest, const WCHAR* source) const;
-    void set_environment_var(TCHAR* dest, const std::string& varStr) const;
-
```

Comment:
It is not a refactoring: see closing comment in #69.
And yes, I remember about tests, just haven't finished them yet.